content/guides/rpa/rpa-use-case-inventory.md

```diff
@@ -10,8 +10,7 @@ guide: rpa
 
 The Federal RPA CoP developed the “RPA Use Case Inventory (UCI)”, which provides detailed information on over 300 RPA Use Cases across the federal government. For each automation, the inventory includes descriptive data such as agency name, business function, a high-level description of the use case, and an estimate of the annualized capacity provided.
 
-{{< button href="https://digital.gov/pdf/federal-rpa-use-case-inventory-compliant.pdf" text="Download the RPA Use Case Inventory" >}}  (PDF, 271.08 KB, 13 pages)
-
+{{< asset-static file="federal-rpa-use-case-inventory-compliant.pdf" label="Download the RPA Use Case Inventory" >}}  (PDF, 271.08 KB, 13 pages)
```

Comment:
This is a button outline on live site. This might be okay for now, but we should consider button variants.

Reply:
Sounds good, can update the shortcode file for returning a button link.